signer/src/bitcoin/utxo.rs

```diff
@@ -1466,6 +1482,104 @@ pub trait TxDeconstructor: BitcoinInputsOutputs {
             .collect()
     }
 
+    /// Return the withdrawal outputs, matching the tx outputs to the decoded
+    /// withdrawal IDs
+    fn to_withdrawal_outputs(
+        &self,
+        tx_outputs: &[TxOutput],
+    ) -> Result<Vec<WithdrawalTxOutput>, Error> {
+        // If the first output is not a SignersOutput, nothing to do
+        match tx_outputs.first() {
```

Comment:
Suggestion: Same comment as below, or could probably use `.map_or()`

Reply:
Done in [6388c9e](https://github.com/stacks-network/sbtc/pull/1509/commits/6388c9e5c9be7efc00b0ee77ecc6921efecf9338)